### PR TITLE
Add Intercom

### PR DESCRIPTION
**Pull request by backspace**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -67,6 +75,12 @@ const App = Application.extend(Evented, {
         return HS.beacon.identify(userParams);
       });
     }
+  },
+
+  setIntercomUser(user) {
+    this.get('intercom').set('user.name', user.name);
+    this.get('intercom').set('user.email', user.email);
+    // FIXME createdAt?
```
Comment (joshk) [inline app/app.js]: we can use `user.first_logged_in_at` i think, and assign it to `createdAt`

Comment (backspace) [reply app/app.js]: 👍🏻 thanks, I’ve added that now. I’m having a hard time getting anything to build locally so I’ve just pushed and will see how it works on the PR deployment.

File: app/services/auth.js [hunk 2]
```diff
@@ -161,7 +161,7 @@ export default Service.extend({
     if (proVersion && get(config, 'intercom.enabled')) {
       this.get('intercom').set('user.name', user.name);
       this.get('intercom').set('user.email', user.email);
-      // FIXME createdAt?
+      this.get('intercom').set('user.createdAt', user.last_synced_at);
```
Comment (joshk) [inline app/services/auth.js]: ohhhhh, no no no, `first_logged_in_at`

Comment (backspace) [reply app/services/auth.js]: oops thanks, I didn’t get enough sleep 😳

Comment (backspace) [reply app/services/auth.js]: I finally got it running locally, looks like it’s `synced_at`:

image

Comment (backspace) [reply app/services/auth.js]: er wait… `first_logged_in_at` doesn’t exist, or anything like it

Comment (backspace) [reply app/services/auth.js]: (for observers: `created_at`, obvsy; like I said, underslept 😆)

**Pull request merged**